nservicebus/messaging/transactions.md

```diff
@@ -23,9 +23,11 @@ Based on transaction handling mode, NServiceBus offers three levels of guarantee
 
 ### Unreliable
 
-In this mode the transport in use does not attempt to wrap the receive operation in any kind of transaction. Once the message has been received, it can't be put back into the queue. Should the message processing fail for any reason (including system crash), the message is **permanently lost**. 
+In this mode the transport in use does not attempt to wrap the receive operation in any kind of transaction. 
 
-NOTE: No retries will be performed in this mode and messages will not be forwarded to the error queue in case of failures
+Once a message has been received, if the message processing fails it should still be put into the error queue. If there is a critical failure, including system or endpoint crash, the message is **permanently lost**. 
```

Comment:
I think it should be the other way around. Document current behavior as is with what we intend to do in V6. 
The behavior change should be in the upgrade guide. 
Since v6 isn't currently released, it might be confusing to someone who's on v5.